### PR TITLE
cli: do not suggest liveness probes for controller owned pods

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1542325

I think it is safe to assume that any Pod with ownerReference.Controller=true is not controlled by a human an therefore we should not suggests users to augment them with probes.

### CODE REVIEW

File: pkg/oc/graph/kubegraph/edges.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ const (
 	ExposedThroughServiceEdgeKind = "ExposedThroughService"
 	// ManagedByControllerEdgeKind goes from Pod to controller when the Pod satisfies a controller's label selector
 	ManagedByControllerEdgeKind = "ManagedByController"
+	// OwnedByControllerEdgeKind indicates that an object is owned by a controller (and owner reference has controller=true)
+	OwnedByControllerEdgeKind = "OwnedByController"
```
Comment (juanvallejo) [inline pkg/oc/graph/kubegraph/edges.go]: Does this edge end up being added anywhere?

Comment (mfojtik) [reply pkg/oc/graph/kubegraph/edges.go]: no, leftover, good catch!

**Comment by mfojtik** [comment]
@juanvallejo removed lefover.

**Comment by kargakis** [comment]
Yes! Finally.

**Pull request merged**